reference/service/saml-okta.md

```diff
@@ -0,0 +1,119 @@
+---
+title: SAML Configuration > Okta
+---
+
+This guide walks through how to configure Okta as a SAML SSO identity provider (IdP) for use with
+the Pulumi Cloud Console.
+
+> The screen shots below are using the Okta _Classic UI_. You can switch to it by clicking the gear
+> icon in the top right of the screen.
+
+## Creating the Okta Application
+
+The first step is to create a new Okta Application Integration. Of the various "sign on methods"
+available, choose "SAML 2.0".
+
+![Creating an Okta Application](../../images/reference/service/saml-okta/create-okta-application.png)
+
+### Configuring the Application
+
+Next you will be guided through a wizard to configure the Okta application. The first step is to
+give it a name, e.g. "Pulumi Cloud Console", and an icon.
+
+![Configuring a SAML Integration](../../images/reference/service/saml-okta/create-saml-integration.png)
+
+The next step is where you configure the SAML application's settings. The values to provide will
+depend on the name of your Pulumi organization, e.g. `robot-co` or `acme`.
+
+> [Contact us](https://www.pulumi.com/about/#contact-us) to provision the SAML-backed Pulumi
```

Comment:
Feels like this should be more prominent, in fact. It should be clearer that users cannot actually self-provision SAML organizations today and that this document is meant for users who have already done that.